apm-sniffer/config/agent.config

```diff
@@ -20,6 +20,8 @@ agent.namespace=${SW_AGENT_NAMESPACE:}
 # The service name in UI
 agent.service_name=${SW_AGENT_NAME:Your_ApplicationName}
 
+agent.instance_properties_json=${SW_INSTANCE_PROPERTIES_JSON:}
```

Comment:
Can you please put this in the same order as the doc (move this to the next line of `agent.instance_name`)